packages/datadog-plugin-next/test/index.spec.js

```diff
@@ -89,11 +96,64 @@ describe('Plugin', function () {
         const files = [
           'package.json',
           'node_modules',
-          '.next'
+          '.next',
+          'yarn.lock'
         ]
         const paths = files.map(file => `${__dirname}/${file}`)
         execSync(`rm -rf ${paths.join(' ')}`)
       })
+    }
+
+    const initStandaloneFiles = () => {
+      before(() => {
+        // insert tracer init into auto-generated server.js
```

Comment:
Is this how users would use this? Shouldn't the real functionality be tested instead (for example using NODE_OPTIONS).

Reply:
See response below, but it was my understanding that some users choose to initialize the tracer at the top of this auto-generated file, but it can be removed in favor of using NODE_OPTIONS